src/mainboard/protectli/vault_adl_n/variants/vp2440/overridetree.cb

```diff
@@ -24,21 +24,23 @@ chip soc/intel/alderlake
 		end
 
 		device ref xhci on
-			register "usb2_ports[0]" = "USB2_PORT_EMPTY"
-			register "usb2_ports[5]" = "USB2_PORT_SHORT(OC_SKIP)" # 5G/LTE
-			register "usb3_ports[1]" = "USB3_PORT_EMPTY"
+			device ref xhci_root_hub on
+				register "usb2_ports[0]" = "USB2_PORT_EMPTY"
+				register "usb2_ports[5]" = "USB2_PORT_SHORT(OC_SKIP)" # 5G/LTE
+				register "usb3_ports[1]" = "USB3_PORT_EMPTY"
```

Comment:
Are you sure register assignments need to be moved?  Original device tree has them at `xhci` device: https://github.com/Dasharo/coreboot/blob/6762d4516b609466f2ea865f7cebca342943a60c/src/mainboard/protectli/vault_adl_n/devicetree.cb#L82-L85

Reply:
right. https://github.com/Dasharo/coreboot/pull/678/commits/2b165dd1c41b3fcfc8014abb4d2a8e4bf119279b